core/metrics/src/pusher.rs

```diff
@@ -54,12 +54,36 @@ pub async fn push_forever(host_url: &str) {
 }
 
 pub async fn push(client: &Client, push_url: String) {
-    let current_metrics = crate::service::export_metrics().await;
+    let metrics = crate::service::export_metrics().await;
     let res = client
         .put(push_url.as_str())
-        .send_body(current_metrics)
+        .send_body(metrics.clone())
         .await;
-    log::trace!("Pushed current metrics {:#?}", res);
+    match res {
```

Comment:
Here I'm improving what we are communicating to the users through the logs.
Until now, noting was communicated (only trace message, which was not displayed most of the times).
That's why we missed it stopped to work.